docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -1,500 +1,594 @@
-Token                                                            Type            Labels   Expiry Time (UTC)
----------------------------------------------------------------- --------------- -------- ---------------------------
-(=presets.tokens.first=)                                 trusted_cluster          28 Apr 22 19:19 UTC (4m48s)
-```
-
-<Details title="Revoking join tokens">
-
-You can revoke a join token with the following command:
-
-```code
-$ tctl tokens rm (=presets.tokens.first=)
-```
-
-</Details>
-
-<Notice
-  type="tip"
->
-
-  It's important to note that join tokens are only used to establish a
-  connection for the first time. Clusters will exchange certificates and
-  won't use tokens to re-establish their connection afterward.
-
-</Notice>
-
-### Define a Trusted Cluster resource
-
-On your local machine, create a file called `trusted_cluster.yaml` with the
-following content:
-
-```yaml
-# trusted_cluster.yaml
-kind: trusted_cluster
-version: v2
-metadata:
-  name: rootcluster.example.com
-spec:
-  enabled: true
-  token: (=presets.tokens.first=)
-  tunnel_addr: rootcluster.example.com:11106
-  web_proxy_addr: rootcluster.example.com:443
-  role_map:
-    - remote: "access"
-      local: ["visitor"]
-```
-
-Change the fields of `trusted_cluster.yaml` as follows:
-
-#### `metadata.name`
-
-Use the name of your root cluster, e.g., `example.teleport.sh`.
-
-#### `spec.token`
-
-This is join token you created earlier.
-
-#### `spec.tunnel_addr`
-
-This is the reverse tunnel address of the Proxy Service in the root cluster. Run
-the following command to retrieve the value you should use:
-
-<Tabs>
-<TabItem scope={["oss", "enterprise"]} label="Self-Hosted">
-
-```code
-$ PROXY=rootcluster.example.com
-$ curl https://${PROXY?}/webapi/ping | jq 'if .proxy.tls_routing_enabled == true then .proxy.ssh.public_addr else .proxy.ssh.ssh_tunnel_public_addr end'
-"rootcluster.example.com:443"
-```
-
-</TabItem>
-<TabItem scope={["cloud"]} label="Teleport Enterprise Cloud">
-
-```code
-$ PROXY=rootcluster.teleport.sh
-$ curl https://${PROXY?}/webapi/ping | jq 'if .proxy.tls_routing_enabled == true then .proxy.ssh.public_addr else .proxy.ssh.ssh_tunnel_public_addr end'
-"rootcluster.teleport.sh:443"
-```
-
-</TabItem>
-
-</Tabs>
-
-#### `web_proxy_addr`
-
-This is the address of the Proxy Service on the root cluster. Obtain this with the
+1. Add the local `visitor` user and create a home directory for the user by running the 
 following command:
+   
+   ```code
+   $ sudo useradd --create-home visitor
+   ```
+   
+   The home directory is required for the `visitor` user to access a shell on the server.
+
+1. Sign out of all user logins and clusters by running the following command:
+   
+   ```code
+   $ tsh logout
+   ```
+
+1. Sign in to your **leaf cluster** from your administrative workstation using 
+your Teleport username:
+   
+   ```code
+   $ tsh login --proxy=<Var name="leafcluster.teleport.sh" /> --user=<Var name="myuser" />
```

Comment:
Earlier in this guide we used example.com and now we've switched to teleport.sh. I would be consistent throughout the guide so that the reader can do some pattern recognition.